packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -223,7 +223,44 @@ class TextStyleTween extends Tween<TextStyle> {
 ///
 /// Which properties are animated is left up to the subclass. Subclasses' States
 /// must extend [ImplicitlyAnimatedWidgetState] and provide a way to visit the
-/// relevant fields to animate.
+/// relevant fields to animate. The "see also" section below lists a number of
+/// concrete subclasses that ship with the framework (their names usually start
+/// with `Animated`).
+///
+/// ## Relationship to [AnimatedWidget]s
+///
+/// [ImplicitlyAnimatedWidget]s (and its subclasses) create and manage their own
+/// internal [AnimationController]s to power the animation. In contrast to that,
+/// [AnimatedWidget]s (and its subclasses) take an explicit [Animation] as
+/// argument to drive the animation. This makes [ImplicitlyAnimatedWidget]s
+/// simpler to use, but it may be less efficient for multiple parallel
+/// animations: Instead of using multiple [ImplicitlyAnimatedWidget]s, which
+/// all instantiate their own [AnimationController] behind the scenes, consider
+/// instantiating one explicit [AnimationController] that is passed to
+/// (subclasses of) [AnimatedWidget]s instead of using (subclasses of)
+/// [ImplicitlyAnimatedWidget]s.
```

Comment:
is it really that much less efficient? i would imagine it's about the same in practice. AnimationControllers have very little overhead.